.env.example

```diff
@@ -1,2 +1,3 @@
 GOOGLE_API_KEY="Sample Key"
 API_HOST=http://localhost:5000
+LOCAL=true
```

Comment:
The issue with this solution is that if the developer doesn't specify LOCAL as true in the env file or the env file doesn't exist, then the port won't be added to the generated url. 
I couldn't figure out any other approach. Open for alternative approach.

Reply:
This is not needed at all. You can just look for `location.port`. If you get a value, use it. Else don't